seismic-app/src/styles/_blog.scss

```diff
@@ -0,0 +1,107 @@
+h1 {
+  font-family: 'Titillium Web', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 37px;
+  text-decoration-line: underline;
+  margin-bottom: 18px;
+}
+
+.blog-wrapper {
+  padding: 32px;
+
+  @media (max-width: 991px) {
+    padding: 24px;
+  }
+
+  @media (max-width: 767px) {
+    padding: 16px;
+  }
+
+  @media (max-width: 575px) {
+    padding: 8px;
+  }
+}
+
+.load-more {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+
+  button {
+    font-family: 'Titillium Web', sans-serif;
```

Comment:
The comments from [line 2](https://github.com/code4romania/seismic-risc-client/pull/20/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.scss#diff-89e8ee054e0753ec52afbc9b716fc4e4R2) also, let's just place this font in a single spot on the page to avoid duplication - it's probably going to make it easier to make sure the font usage is uniform

Reply:
same as line 2, resolving conversation